docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -63,95 +99,69 @@ An example of the `mbed_app.json` file:
-
 #### Thread commissioning
 
-By default, the Thread application uses the static network link configuration defined in the [mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
+By default, the Thread application uses the static network link configuration defined in the [Mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
 If you want to commission a Thread device, see [how to commission a Thread device in practice](../reference/thread-tech.html#thread-commissioning).
 
-The Thread stack learns the network settings from the commissioning process and saves them to RAM memory. Therefore, the learned network settings are lost when you restart the device next time. To prevent re-commissioning, you can save the Thread configuration settings to an SD card as follows (only in `K64F`):
+The Thread stack learns the network settings from the commissioning process and, by default, saves them to RAM memory. Therefore, the learned network settings are lost every time you restart the device. To prevent the need to recommission, save the Thread configuration settings to an SD card instead of RAM (only for the `K64F`):<!--why? can't it work with other boards that have an SD card?-->
 
 - Change `storage-device` to `MESH_NVM_SD_CARD` in the `./configs/mesh_thread.json` file.
-- Enable commissioning as descibed in the referred instructions.
+- Enable commissioning as described in the referred instructions.<!--there are two links in the above paragraph; spare people rereading and give them a link here, too-->
 - Compile and program the application.
 - Insert an erased or FAT-formatted SD card to the `K64F` memory card slot. The application will initialize the SD card with the appropriate file system on first use.
 - Commission the device to the Thread network.
-- When you restart the device next time, the device reads the Thread configuration settings from the SD card and tries to attach to an existing network.
+- When you restart the device next time, the device reads the Thread configuration settings from the SD card and tries to connect to an existing network.
+
+## Security-based hardware requirements
 
-## Requirements for hardware
+The networking stack in this example requires TLS functionality. On devices where hardware entropy is not present, TLS is disabled by default. This results in compile time failures or linking failures.
 
-The networking stack in this example requires TLS functionality to be enabled on Mbed TLS. On devices where hardware entropy is not present, TLS is disabled by default. This results in compile time failures or linking failures.
+To learn why entropy is required, read the [TLS porting guide](../porting/entropy-sources.html).<!--the first sentence said "TLS functionality", but the next two references were specifically for hardware entropy. which is correct?-->
```

Comment:
@SeppoTakalo @jarlamsa Can you answer this?

Reply:
Mbed TLS uses the hardware entropy to provide reliable RNG for application, that is the relation between those terms.